backend/app/graphql/onboarding_request.py

```diff
@@ -31,7 +28,103 @@ class OnboardingRequest(graphene.ObjectType):
     status = graphene.String()
 
 
+# Queries
+
+# Return object for queries
+class GetOnboardingRequest(Query):
+    contact_name = graphene.String()
+    contact_email = graphene.String()
+    contact_phone = graphene.String()
+    role = graphene.String()
+    date_submitted = graphene.DateTime()
+    status = graphene.String()
+
+
+class OnboardingRequestQueries(QueryList):
+    getAllOnboardingRequests = graphene.List(
+        GetOnboardingRequest,
+        first=graphene.Int(default_value=5),
+        offset=graphene.Int(default_value=0),
+        role=graphene.String(default_value=""),
+        status=graphene.String(default_value=""),
+    )
+    getOnboardingRequestById = graphene.List(
+        GetOnboardingRequest,
+        id=graphene.String(),
+        first=graphene.Int(default_value=5),
+        offset=graphene.Int(default_value=0),
+    )
+
+    def resolve_getAllOnboardingRequests(self, info, first, offset, role, status):
+        requests = services["onboarding_request_service"].get_all_onboarding_requests()
+        if role != "":
```

Comment:
Would be best to do filtering in the service, since it should be "frontend"-agnostic (i.e., if we were to switch to a framework other than GraphQL, not that we would).

Also, doing filtering in the service will allow you to make use of more efficient MongoDB filters instead of doing this sort of manual filter. See my next comment!